lib/FrontendTool/ReferenceDependencies.cpp

```diff
@@ -402,24 +395,138 @@ bool swift::emitReferenceDependencies(DiagnosticEngine &diags, SourceFile *SF,
     out <<  mangleTypeAsContext(i->first.first);
     out << "\"\n";
   }
+}
 
-  out << "depends-dynamic-lookup:\n";
-  for (auto &entry : sortedByName(tracker->getDynamicLookupNames())) {
+static void writeExternalDynamicLookupDependencies(
```

Comment:
No "External" needed here (that refers to things outside the current module).

Reply:
Fixed.